[Blog Post] 0.36: Headless JS, the Keyboard API, & more

hramos opened the pull request:
This is a draft for an upcoming blog post scheduled for October 24, 2016. Do not merge into `master` before that date.

lacker commented on blog/2016-10-24-whats-new-in-react-native.md:
```diff
@@ -0,0 +1,86 @@
+
+## Keyboard Component
+
+Working with the on-screen keyboard is now easier with [`Keyboard`](/react-native/docs/keyboard.html). You can now listen for native keyboard events and react to them. For example, to dismiss the active keyboard, simply call `Keyboard.dimiss()`:
```
dimiss -> dismiss

hramos replied:
Ah, silly mitake.

lacker commented on blog/2016-10-24-whats-new-in-react-native.md:
```diff
@@ -0,0 +1,86 @@
+
+Using Headless JS does require some native Java code to be written in order to allow you to start up the service when needed. Take a look at our new [Headless JS docs](/react-native/docs/headless-js-android.html) to learn more!
+
+## Keyboard Component
```
Is this a react component? If not, how about callign this section "The Keyboard API"

hramos commented on blog/2016-10-24-whats-new-in-react-native.md:
```diff
@@ -0,0 +1,86 @@
+
+Read on to learn more about what's new in React Native v0.36.
+
+## Headless JS
```
It is, good catch.

lacker commented:
How about naming it something SEO'd, like: "0.36: Headless JS, the Keyboard API, & more"

hramos commented:
Addressed all feedback. I also moved the upgrade instructions to the bottom of the post as it makes for a nicer preview (the first paragraph is displayed in the blog index and whenever the post is shared on social media).

caabernathy commented:
How about mentioning Apple TV support (talked about here https://medium.com/@7ynk3r/react-native-apple-tv-today-48beb398a1ab) if it is indeed new in this release.

hramos commented:
If Apple TV support is ready for use, we should mention it here. I thought it was still a WIP though.

hramos commented:
We also should make sure Apple TV is properly documented before blogging about it. I vote we write a separate post when docs are ready, it's big enough news on its own.

lacker commented:
Yeah let's write about Apple TV separately - the stuff in this release is not the final stuff on that.

hramos commented:
@facebook-github-bot shipit-ninja
